chia/full_node/full_node.py

```diff
@@ -248,6 +253,7 @@ def sql_trace_callback(req: str):
         self.transaction_responses: List[Tuple[bytes32, MempoolInclusionStatus, Optional[Err]]] = []
 
         self.weight_proof_handler = None
+        self.weight_proof_handler_v2 = None
```

Comment:
Ah i see, since we still need to return the old ones. I think it's OK to do it this way then, but can we add a comment explaining that both are needed because we support returning both versions of the weight proof?